blocks/deal/deal.css

```diff
@@ -0,0 +1,47 @@
+.deal.block {
+    padding: var(--spacing-l);
+    background-color: var(--footer-background-color);
+}
+
+.deal.block h2 {
+    font-weight: var(--font-weight-bold);
+}
+
+.deal.block > div:nth-child(1) > div {
+    display: flex;
+}
+
+.deal.block > div:nth-child(1) > div p {
+    margin: 0;
+}
+
+.deal.block > div:nth-child(1) > div img {
+    height: 40px;
+    margin-right: 40px;
```

Comment:
should we create a new spacing var and use it here?

Reply:
@cuzuco2 i'm not sure - i usually crate a var when i know this value is shared somewhere else. if it is not, it would leak into the globals for no reason.